Makefile

```diff
@@ -0,0 +1,84 @@
+GO ?= go
+EPOCH_TEST_COMMIT ?= f2925f58acc259c4b894353f5fc404bdeb40028e
+PROJECT := github.com/kubernetes-incubator/cri-containerd
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
+GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
```

Comment:
The branch is not used. Let's remove it and add it back when we add the `version` support.